1/2_add_transport.md

```diff
@@ -1,9 +1,53 @@
-* A little about which transports are we going to use and there implementations.
-* Wrtc star why do we need to declare
-* your turn
-    - Install exact versions of transports: "libp2p-tcp": "^0.13.0", "libp2p-webrtc-star": "^0.16.1", "libp2p-websockets": "^0.12.2", "wrtc": "^0.4.1",   "libp2p-mdns": "^0.12.3", provide the npm command to do so
-    - Include transport, declare webrtc star
-    - modify option.modules.transport to add a list of transports
+## Adding Transports
+
+### Let's add some transport protocols! 
+Welcome back to the next segment of this chapter. In the previous segment of this worship, we 
+1. imported `libp2p`,
+2. created an empty map named `options`, and 
+3. created a libp2p node inside our `main` function. 
+
+In this segment, we'll actually go ahead and add some cool transport protocols in our `awesome-p2p-chat` application. Now as we discussed earlier in the introduction to transports segment, libp2p provides an amazing functionality allowing you to use multiple transports in the same application and that is what we will be doing today!
+
+So without any further ado, lets go ahead and start discussing and implementing the protocols that we'll be using.
+
+### Various Transport Protocols, we'll be using
+
+* `libp2p-tcp` : This is the libp2p implementation of the standard TCP (Transfer Control Protocol). 
+
+	*The basic function of  **TCP** is to segment the information into smaller data packets, index them, and add additional information such as source & destination.* 
+
+	More information about TCP is given in the the [first section of this workshop](http://salabs.me/jslibp2p-chat-tutorial/#/1/introduction). It could be simply installed using the command:
+	```bash
+	npm i libp2p-tcp@0.13.0
```

Comment:
@Xavian-Brooker is this the correct version also with the version bump? Check package.json in the asset folder